src/views/Home/components/chapelProgress/index.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { Doughnut } from 'react-chartjs-2';
+import Card, { CardContent } from 'material-ui/Card';
+
+import { gordonColors } from '../../../../theme';
+import user from '../../../../services/user';
+import GordonLoader from '../../../../components/Loader';
+
+export default class ChapelPsrogress extends Component {
+  constructor(props) {
+    super(props);
+
+    this.loadChapel = this.loadChapel.bind(this);
+
+    this.state = {
+      loading: true,
+      chapelCredits: {},
+    };
+  }
+  componentWillMount() {
+    this.loadChapel();
+  }
+  async loadChapel() {
+    const chapelCredits = await user.getChapelCredits();
+    this.setState({ loading: false });
+    this.setState({ chapelCredits });
```

Comment:
This line can be combined with the previous line:
`this.setState({ chapelCredits, loading: false });`